libraries/Microsoft.Bot.Schema/Context/InputContext.cs

```diff
@@ -0,0 +1,62 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using Newtonsoft.Json;
+
+namespace Microsoft.Bot.Schema
+{
+    /// <summary>
+    /// Input context for the next turn.
+    /// </summary>
+    public partial class InputContext
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="InputContext"/> class.
+        /// </summary>
+        /// <param name="locale">Expected locale.</param>
+        /// <param name="expected">Expected intents and entities.</param>
+        /// <param name="possible">Possible intents and entities.</param>
+        public InputContext(string locale = null, RecognizerDescription expected = null, RecognizerDescription possible = null)
+        {
+            Locale = locale;
+            Expected = expected ?? new RecognizerDescription();
+            Possible = possible ?? new RecognizerDescription();
+        }
+
+        /// <summary>
+        /// Gets the expected locale.
+        /// </summary>
+        /// <value>Locale.</value>
+        public string Locale { get; }
```

Comment:
Does this need to be here?  How does this relate to activity.locale, turn.locale, user.locale, etc?

Reply:
This is the expected locale of the input which priming needs and is the simplest kind of priming.  It is usually the dialog context locale, but happy to use another one.  There was also some desire to have a self-contained package.